### ISSUE TITLE
White matter hyperintensity in posterior limbs

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#9731]
**Preferred term label:**
White matter hyperintensity in posterior limbs

**Synonyms**

**Definition (free text, please give PubMed ID)**
Areas of brighter than expected signal on magnetic resonance imaging emanating from the posterior limbs of internal capsule (PMID: 30762766; 30612693). Comment - The internal capsule is a subcortical white matter structure situated in the inferomedial portion of each cerebral hemisphere. The posterior limb of internal capsule is the portion of the internal capsule posterior to the genu of the internal capsule.

**Parent term (use hpo.jax.org/app)**
Hyperintensity of cerebral white matter on MRI (HP:0030890)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
@jagcsun  -- could you check if this feature is specific to T1 or T2 MRI imaging?
